src/modules/brand/components/TransferTokenDialog/TransferTokenDialog.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, Dialog, Grid, IconButton, Typography } from '@material-ui/core';
+import classNames from 'classnames';
+import { NftType } from 'modules/createNFT/actions/createNft';
+import { t } from 'modules/i18n/utils/intl';
+import { useMemo, useCallback } from 'react';
+import { Button } from 'modules/uiKit/Button';
+import { Field, Form, FormRenderProps } from 'react-final-form';
+import { InputField } from 'modules/form/components/InputField';
+import { FormErrors } from 'modules/form/utils/FormErrors';
+import { useTransferTokenDialogStyles } from './useTransferTokenDialogStyles';
+import { AngleDownIcon } from 'modules/common/components/Icons/AngleDownIcon';
+import { AngleUpIcon } from 'modules/common/components/Icons/AngleUpIcon';
+import { CloseIcon } from 'modules/common/components/Icons/CloseIcon';
+
+const MIN_QUANTITY = 1;
+
+export interface ITransferFormValues {
+  toAddress: string;
+  quantity: number;
+}
+
+export const TransferTokenDialog = ({
+  loading,
+  isOpen,
+  standard,
+  maxQuantity,
+  onClose,
+  onSubmit,
+}: {
+  loading: boolean;
+  isOpen: boolean;
+  standard: NftType;
+  maxQuantity?: number;
+  onClose: () => void;
+  onSubmit: (value: ITransferFormValues, form: any, callback: any) => void;
+}) => {
```

Comment:
Better to move the typing into a named interface IBurnTokenDialogProps like other copmonents in the project.